convex/schema.ts

```diff
@@ -2,5 +2,10 @@ import { defineSchema, defineTable } from "convex/server";
 import { v } from "convex/values";
 
 export default defineSchema({
-  input: defineTable({ text: v.string() }),
+  slugs: defineTable({
+    slug: v.string(),
+    // startTime: v.string(),
+    passedTests: v.optional(v.number()),
```

Comment:
I think making it non optional is more reliable.
If you open a slug without typing, it will save it to the leaderboard but without values,